src/xdpapi/afxdp.c

```diff
@@ -7,7 +7,6 @@
 #include <assert.h>
 
 HRESULT
-XDPAPI
```

Comment:
Why remove these?

Reply:
That macro expands out to `__declspec(dllexport)` which we no longer need.

Reply:
(We still use that macro for the XdpOpenApi/XdpCloseApi routines)